### DISCUSSION TITLE
Students developer pack

### DISCUSSION THREAD

**Discussion by itzenoughabhi** [community/community#131430] [Accessibility]
### Hi everyone,

I am not getting students developer pack 
It showing available soon from long time 
Help me with this

**Comment by trikrapka** [comment]
> ### Hi everyone,
> I am not getting students developer pack It showing available soon from long time Help me with this

It takes time, just wait. It took like 20 days for me, and it can possibly take even more time =(

**Comment by itzenoughabhi** [comment]
done
